package.json

```diff
@@ -13,10 +13,11 @@
   },
   "dependencies": {
     "babel-polyfill": "^6.26.0",
-    "vue": "^2.5.16",
+    "vue": "2.6.10",
```

Comment:
build conflict, dependency build with 2.6.10 where we were building with 2.6.12?